packages/components/src/ui/create-styles/create-style-system/constants.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Uses the the prefix for the CSS variables compiled by the system.
+ */
+export const NAMESPACE = '--wp-g2';
```

Comment:
this should probably be `--wp-admin` like our existing CSS variables but maybe we can keep this temporary prefix until we remove the css variables.

Reply:
Yes, let's put that off until we've culled the number of variables, it'll make it easier to make the distinction when debugging.